From c10cb2742af6aef3ae6ee6d538a6a91e5b0c9078 Mon Sep 17 00:00:00 2001
From: Keerthana K <keerthanak@vmware.com>
Date: Fri, 18 Nov 2022 07:12:24 +0000
Subject: [PATCH 1/2] stalld: Include 'FF' and 'CG' config params in service
 file

Add FIFO and config granularity options to service file

Signed-off-by: Keerthana K <keerthanak@vmware.com>
---
 redhat/stalld.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/redhat/stalld.service b/redhat/stalld.service
index b1988cc..fdebf5a 100644
--- a/redhat/stalld.service
+++ b/redhat/stalld.service
@@ -9,7 +9,7 @@ ExecStartPre=/usr/bin/throttlectl off
 
 # In case the regex passed to IT or IP includes C escape sequences,
 # use ${IT} or ${IP} instead of $IT or $IP
-ExecStart=/usr/bin/stalld --systemd $CLIST $AGGR $BP $BR $BD $THRESH $LOGGING $FG $PF $IT $IP $VB
+ExecStart=/usr/bin/stalld --systemd $CLIST $AGGR $BP $BR $BD $THRESH $LOGGING $FG $PF $IT $IP $VB $FF $CG
 ExecStopPost=/usr/bin/throttlectl on
 CPUSchedulingPolicy=fifo
 CPUSchedulingPriority=10
-- 
2.28.0

